pages/who-we-are-hiring.md

```diff
@@ -2,33 +2,70 @@
+
+**Data Architect**
+Data architects on the 18F Consulting team,  help agencies liberate and embrace their data, equip them with profound insights drawn from that data, empowering them to more effectively fulfill their mission. 
 
-## Upcoming positions
-We're currently revamping our application process. When it relaunches in mid-November, we will be hiring for positions **including but not limited to** the following:
+**Practical Dev**
 
-###DevOps engineer
-Our DevOps engineers work tirelessly to arm developers with the best tools and ensure system uptime and performance. They're comfortable with bridging gaps with legacy development or operations teams.
+Practical Devs on the 18F consulting team provide broad development expertise and use their consultative know-how to help agencies by conducting workshops and protosketching sessions to build out proofs-of-concept and prototypes
+
+### Delivery Team
+
+**DevOps & Site Reliability Engineer**
+
+DevOps engineers work tirelessly to arm developers with the best tools and ensure system uptime and performance, helping to bridge gaps with legacy development or operations teams.
+
+**Product Lead**
+
+18F Product Lead is responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients).
+
+**Software Developer (Front-End, Back-End, Full-Stack)**
 
-###Software development (front end)
 Using modern, open source web frameworks and tools, our front and back end developers work in small, cross-functional teams to produce functional, scalable web services.
 
-###Visual and interaction design
-Our designers create wireframes, HTML prototypes, high-fidelity mockups, and more, based on a firm understanding of user needs. They do what it takes to communicate (and sometimes implement) these solutions.
+### Design Team
+
+**Content Designer**
+
+Content designers are responsible for creating internal and external content of all varieties and organizing that content for maximum user benefit.
 
-###Product management (DC-based)
-Product leads work closely with agency partners to define and prioritize product value in agile cycles. They remove blocks for the team, create and groom backlogs, champion user testing, and ensure we're always shipping valuable code.
+**Front End Designer**
+
+Front end designers work with user experience and visual designers, content creators and development teammates to improve, refine, and develop a well-structured front end workflow and culture.
+
+**User Experience Designer**
+
+User experience designers  research, derive insights, generate concepts, communicate those concepts visually, and work with developers and product managers to build them.
+
+**Visual Designer**
+
+Visual designers at 18F craft highly usable and beautiful web-based experiences for the benefit of the public.
+
+## Who is eligible to work at 18F
```

Comment:
@fureigh Sure. The only reason I moved them down was because we will be referring applicants  from the new essay on the application to use the roles on this page. I feel like ideally the roles would be on a separate page anyway so this is just MVP.